test/js/node/cluster/cluster.test.ts

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from "bun:test";
+import path from "node:path";
+import "harness";
+
+function new_test(name: string) {
+  test(name, () => {
+    expect([path.join(import.meta.dir, "fixtures", name)]).toRun();
+  });
+}
+
+new_test("cwd.ts");
+new_test("call-and-destroy.ts");
+new_test("worker-constructor.ts");
+new_test("worker-death.ts");
+new_test("worker-events.ts");
```

Comment:
Where are tests that check that responding to HTTP requests works?

Reply:
added now